src/js_src/components/disease/index.js

```diff
@@ -1,224 +1,44 @@
 import React, { Component } from 'react';
-import DataSourceLink from '../dataSourceLink';
+import FlybaseDataGrid from 'react-flybase-datagrid';
+import './agr.css';
```

Comment:
I could be wrong, but I don't think this is actually doing anything unless you say something like `import agr from './agr.css';` and then use the `agr` variable to apply `className`s in the JSX. 

But maybe that's a moot point. If it looks fine as is, then maybe we can just do away with the `agr.css` file since it's not really doing anything.